test/Dialect/SV/hw-extract-test-code.mlir

```diff
@@ -94,3 +94,28 @@ module attributes {firrtl.extract.assert =  #hw.output_file<"dir3/", excludeFrom
     }
   }
 }
+
+// -----
+// Check names given to instantations of extracted module, and module names
+
+// CHECK-LABEL: @InstanceName_assert
+// CHECK-LABEL: @InstanceName_assume
+// CHECK-LABEL: @InstanceName_cover
+
+// CHECK-LABEL: @InstanceName
+// CHECK-NOT: InvisibleBind
```

Comment:
Checking explicitly that this is not named `InvisibleBind` feels a bit janky. I think it would be better to pattern match the `CHECK-LABEL`s above (and you may have to not use a label there) and then check that that is used in the instantiation name. Alternatively, it's probably fine to just check `hw.instance "[[name:.+]]_assert" @[[name]]`.